libs/barista-components/table/src/selection/selection.ts

```diff
@@ -28,6 +28,13 @@ import {
 } from '@angular/core';
 import { isNumber } from '@dynatrace/barista-components/core';
 
+// HACK: this is required for the Bazel build to work
+// @ts-ignore
+import type { Observable } from 'rxjs';
+// @ts-ignore
+// tslint:disable-next-line
+import type { SelectionChange } from '@angular/cdk/collections';
+
```

Comment:
@lukasholzer Could you help me investigate the issue here? If I don't import `Observable` and `SelectionChange`, I get the following error:

```
ERROR: /Users/rene.buchmayer/repos/barista/libs/barista-components/table/BUILD.bazel:7:22: Compiling Angular templates (ViewEngine - devmode) //libs/barista-components/table:compile failed (Exit 1)
libs/barista-components/table/src/selection/selection.ts:70:12 - error TS2742: The inferred type of 'selectionChange' cannot be named without a reference to '../../../../../external/npm/node_modules/@angular/cdk/collections'. This is likely not portable. A type annotation is necessary.

70   readonly selectionChange = this._selectionModel.changed.asObservable();
              ~~~~~~~~~~~~~~~
```